apps/website-proxy/src/nginx.template.conf

```diff
@@ -49,6 +49,12 @@ http {
             proxy_pass https://website-25-production.k8s.bluedot.org;
         }
 
+        # Redirect 404 errors to the same route on the new website
```

Comment:
Hi @Will-Howard, I think this is solid. 

An idea would be to move the `error_page` call into `location / {`:

```nginx
        location / {
            proxy_ssl_name bluedot.org;
            proxy_set_header Host bluedot.org;
            proxy_pass https://45.76.132.116;
            error_page 404 = @website_25_404;
        }

        location @website_25_404 {
            proxy_pass https://website-25-production.k8s.bluedot.org;
        }
```

Looks a little bit cleaner maybe? What do you think?

Reply:
Yep good shout, I've changed it to this